servicemesh-install/topics/install-supported-config.adoc

```diff
@@ -10,6 +10,7 @@
 The following are the only supported configurations for the {ProductName} {ProductVersion}:
 
 * Red Hat {product-title} version 3.11.
+* Red Hat {product-title} version 4.0 Beta.
```

Comment:
@vikram-redhat, what do you think about adding 4.0 beta references to the 3.x stream?

Reply:
@kalexand-rh My understanding is that the OCP 4 docs are not public so our only option was to include this through the 3.11 docs assuming we continue to publish as part of the OpenShift docs.

Reply:
IIRC this was actually the approach suggested by the OpenShift docs team when we raised the issue of there being no public OCP 4 docs

Reply:
@knrc, I checked with Ali, and you're right that it's fine for now. It sounds like the references will be removed at some point after the OCP 4 docs go public.

Reply:
Fantastic, thanks very much for confirming

Reply:
Absolutely!